test-run.py

```diff
@@ -163,12 +161,22 @@ def main_loop_consistent(failed_test_ids):
         for task_id in task_ids:
             short_status = worker.run_task(task_id)
             if short_status == 'fail':
+                reproduce_file_path = get_reproduce_file(worker.name)
+                if os.path.exists(reproduce_file_path):
+                    with open(reproduce_file_path, 'r') as reproduce_file:
+                        reproduce_content = reproduce_file.read()
+                        color_stdout('Reproduce file %s\n' %
+                                     reproduce_file_path, schema='error')
+                        color_stdout(reproduce_content, schema='serv_text')
+                        color_stdout('\n', '-' * 75, "\n", schema='separator')
```

Comment:
The first '\n' is redundant, because a test file always ends with the newline symbol (by the definition).

`"\n"` differs from any surrounding string by quote type (feels as inconsistency).

Don’t sure the separator (75 dashes) is needed here. Maybe it would be better to use yaml document syntax to visual highlight the file content. See as tarantool itself format a console output:

```
tarantool> 1, 2, 3
---
- 1
- 2
- 3
...
```

It starts with three dashes and ends with three periods.